src/add-to-app/android/project-setup.md

```diff
@@ -293,6 +293,44 @@ dependencies {
   releaseImplementation 'com.example.flutter_module:flutter_release:1.0'
 }
 ```
+</br>
+
+##### Kotlin DSL
```

Comment:
We should probably also change what is currently [line 269](https://github.com/flutter/website/blob/a133bb3be86e4f501eb49f6839b5b35bf77faded/src/add-to-app/android/project-setup.md?plain=1#L269) to indicate that the steps will differ for Java and Kotlin projects.